SFDigitalServices/sf-dahlia-web
# fix: DAH-1900 console warnings

DAH-1900

Waiting on approval and release of Bloom UIC PR: https://github.com/bloom-housing/ui-components/pull/147

Review instructions
You may have some issues with warnings showing up in PR env, so you can also test locally
- Go to directory page
- Open inspector
- Make note of any console warnings (May only appear when running locally)
- Do the same for listing details
